code/game/gamemodes/objective.dm

```diff
@@ -969,3 +969,186 @@ GLOBAL_LIST_EMPTY(possible_items_special)
 	var/area/target_area = get_area(target)
 
 	return (istype(user_area, dropoff) && istype(target_area, dropoff))
+
+////////////////////////////////
+//  Minor traitor Objectives  //
+////////////////////////////////
+/**
+  * # Minor Objectives
+  *
+  * Objectives given to traitors without hijack or martyr, just small flavour things that shouldn't be too hard or impactful.
+  */
+/datum/objective/minor
+
+/**
+  * Sets up the achievement, returns TRUE if it was set up successfully and can be used, FALSE otherwise
+  */
+/datum/objective/minor/proc/finalize()
+	return FALSE
+
+
+/**
+  * # Delete your security records
+  *
+  * Just delete the original record. For now, you don't have to prevent other records being made of your identity
+  */
+/datum/objective/minor/secrecords
+	name = "Delete your security records."
+	explanation_text = "Delete your Nanotrasen security records."
+	/// The security record that needs to be deleted
+	var/datum/data/record/record
+
+/**
+  * Search through all the security records, and find ours.
+  */
+/datum/objective/minor/secrecords/finalize()
+	for(var/datum/data/record/s in GLOB.data_core.security)
+		if(s.fields["name"] == owner.name)
+			record = s
+			return TRUE
+	return FALSE
+
+/**
+  * Returns TRUE if there's no record, returns FALSE otherwise
+  */
+/datum/objective/minor/secrecords/check_completion()
+	return !record || !(record in GLOB.data_core.security)
+
+
+/**
+  * # Kill Pet
+  *
+  * Kill Pet
+  */
+/datum/objective/minor/pet
+	name = "Kill Ian."
+	explanation_text = "Assassinate the HoP's assistant, Ian."
+	/// Pet
+	var/mob/Ian
+
+/**
+  * Chooses and finds pet.area
+  *
+  * Choices are: Ian, Runtime, Anadear, Pun pun, Poly, Renault, Autumn.
+  */
+/datum/objective/minor/pet/finalize()
+	var/list/pets = list("ian", "runtime", "frank", "pun-pun", "poly", "renault", "kalo")
```

Comment:
Add Walter too.
Also, don't think pun-pun is the best choice since he dies almost every round.